### ISSUE TITLE
[Word Desktop]: NVDA is not reciting the link address as well as it is skipping the Image links.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8277]
### Steps to reproduce:
1. Open Sample doc
2. Use NVDA + Space and go to browse mode
3. Use "K" to navigate link by link.
4. Observe that while link navigation image links are getting skipped. 
5. NVDA is announcing only display text for non-image link. It is not announcing the link address.

### Expected behavior:
NVDA should announce Display text as well as link address.

### Actual behavior:
NVDA is not reciting the link address. While navigation it is skipping the image links in the doc.

### System configuration:
NVDA version: 2018.1.1
App Name: Word

NVDA Installed or portable: Installed

**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48

Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by WiproPSAccTest** [comment]
![link reciting](https://user-images.githubusercontent.com/22956130/40101407-f237d98c-5904-11e8-8081-ef237d052652.png)

**Comment by Brian1Gaff** [comment]
I've noticed this at times as well. However I was not sure if the effect was 
the desired effect or not. Some hidden links are impossible to see without 
cutting and pasting to some other software.

**Comment by ehollig** [comment]
I assume this was accidentally closed.

**Comment by Adriani90** [comment]
I cannot reproduce this issue, the example document is missing. Also note that if the link address is not visible for a sighted person, NVDA is not expected to read it. If you add a text to a link and hide the address, then the expected behavior is to reoprt only the visible text without the hidden address.
Regarding image links, you can navigate to them by pressing g for graphics if I am not wrong.

I am closing this issue as works for me. Please comment on this if you have another opinion and we can reopen the discussion.
